solana-programs/anchor/audius-data/scripts/install-dev-dependencies.sh

```diff
@@ -30,6 +30,7 @@ npm i -g "@project-serum/anchor-cli@$ANCHOR_CLI_VERSION"
 anchor --version
 # install dependencies
 yarn install
+chown -R $(whoami) . # needed to resolve some weirdness...
```

Comment:
would you mind making a linear to track figuring out why this happened? super weird

Reply:
[done](https://linear.app/audius/issue/ASI-1059/find-out-why-you-must-chown-deps-in-anchor-audius-data-to-proceed)